app/src/main/java/com/example/dictionary/main/DictionaryState.kt

```diff
@@ -0,0 +1,27 @@
+package com.example.dictionary.main
+
+import com.example.dictionary.domain.model.ErrorInfo
+import com.example.dictionary.data.model.MeaningResponse
+
+data class DictionaryState(
+    val state: UiState = UiState.Empty,
+    val word: String = ""
+)
+
+sealed class UiState {
+    data class Success(
+        val audioPath: String? = null,
+        val meanings: List<MeaningResponse> = emptyList(),
```

Comment:
UiState should never contain a Reposnse, only a Model (for example MeaningModel) or another StateClass